pubspec.yaml

```diff
@@ -14,7 +14,7 @@ dependencies:
     sdk: flutter
   crypto: ^3.0.2
   hex: ^0.2.0
-  http: ^0.13.5
+  http: '>=0.13.5 <=1.1.0'
```

Comment:
Thanks for making this fix too.

In my PR I made the version on any so that there would be no errors with older versions of http. My project uses version 1.2.2, you have a limitation on version 1.1.0.